### ISSUE TITLE
Leaderboard data evaluation code share

### ISSUE THREAD

**Issue by Juhung** [LISTatSNU/FastMRI_challenge#8]
Evaluation.py 를 share 해드립니다.

해당 코드는 Leaderboard data를 테스트할 수 있는 코드이며, test.py 와 비슷합니다.
Leaderboard data를 Data 폴더 안에 넣고 실행하시면 됩니다

[evaluate.zip](https://github.com/LISTatSNU/FastMRI_challenge/files/6846367/evaluate.zip)

**Comment by PyewDegul** [comment]
안녕하세요. 코드 중

print(args.forward_dir)

이 test_part의 Valloss 값을 내보내는데, 이를 Leaderboard의 SSIM값으로 역산하여 계산하면 되는 것인가요?

**Comment by Juhung** [comment]
해당 코드는 경로만을 출력하며, Loss와는 관련없는 내용인 것 같습니다. 
i.e.) "../result/test_Unet/reconstructions_forward"

**Comment by Juhung** [comment]
해당 코드는 경로만을 출력하며, Loss와는 관련없는 내용인 것 같습니다.
i.e.) "../result/test_Unet/reconstructions_forward"

**Comment by HanSooKwan** [comment]
Evaluation 시에 args.input_key를 'image_grappa'로 설정해도 되는건가요? (Test시에도 image_grappa를 input으로 쓸지, image_input을 input으로 쓸지 제가 선택할 수 있는건가요?)

**Comment by HanSooKwan** [comment]
나눠주신 evaluate.py 코드에서 기존 test.py와 달라진게 test data path밖에 없는 것 같고, training 시 val loss만 출력되고 test data에 대한 loss는 출력되지 않는 문제는 해결되지 않은것 같은데, 확인해주실 수 있나요?

**Comment by Juhung** [comment]
네  'image_grappa' 로  설정하셔도  됩니다. h5 파일안에 있는 정보들을 불러와서 목적에 알맞게 사용하시면 될 것 같습니다.

training 에는 training data와 validation data만을 사용하기 때문에, test data에 대한 loss 는 출력되지 않는 것이 당연합니다.

**Comment by HanSooKwan** [comment]
감사합니다!! 제가 한 두 번째 질문이 모호하여 다시 물어보겠습니다. 제 뜻은 test data를 이용해 evaluate.py를 진행했을 'training했을 때 최소 val loss 및 그 때의 epoch' 만 출력이 되고, image_Leaderboard 파일 내에 있는 데이터들 (brain_test1~brain_test50)에 대한 SSIM 값이나 SSIMloss 값이 출력이 되지 않는다는 뜻이였습니다. 이 부분은 저희가 직접 test part를 고쳐서 구현해야 하나요?

**Comment by Juhung** [comment]
맞습니다. 올려드린 코드는 예시를 보여준 것이고, 그 이외에는 직접 구현하시면 됩니다
